src/Program.fs

```diff
@@ -139,18 +157,23 @@ let readConfig (file: string) =
                         then false
                         else parsedJson.["emitMetadata"].ToObject<bool>()
 
-                    Ok {
-                        schema = string parsedJson.["schema"]
-                        queries = fullQueriesPath
-                        project = string parsedJson.["project"]
-                        output = fullOutputPath
-                        errorType = errorType
-                        target = target
-                        createProjectFile = createProjectFile
-                        overrideClientName = overrideClientName
-                        copyLocalLockFileAssemblies = copyLocalLockFileAssemblies
-                        emitMetadata = emitMetadata
-                    }
+                    if serializer = SerializerType.System && target = OutputTarget.Fable ||
+                       serializer = SerializerType.System && target = OutputTarget.Shared then
```

Comment:
if `target = shared` && `serializer = SerializerType.System` it is still valid: only the generated dotnet client will have STJ, not the fable one

Reply:
We need to implement a corresponding converter for Fable similar to your converter to allow this scenario. That is why it is not allowed yet. @Zaid-Ajaj, am I wrong?